test/test.sh

```diff
@@ -1,30 +1,71 @@
-#!/bin/bash -e
```

Comment:
you can remove these changes from this PR.  since they don't overlap we can merge them independently

Reply:
yeah i can rebase them on master